content/docs/introducing-jsx.md

```diff
@@ -87,37 +94,37 @@ function getGreeting(user) {
 }
 ```
 
-### Specifying Attributes with JSX {#specifying-attributes-with-jsx}
+### Attributen specificeren met JSX {#specifying-attributes-with-jsx}
 
-You may use quotes to specify string literals as attributes:
+Je kunt aanhalingstekens gebruiken om string literals als attributen te specificeren:
 
 ```js
 const element = <div tabIndex="0"></div>;
 ```
 
-You may also use curly braces to embed a JavaScript expression in an attribute:
+Je kunt ook accolades gebruiken om een JavaScript expressie in een attribuut te plaatsen:
 
 ```js
 const element = <img src={user.avatarUrl}></img>;
 ```
 
-Don't put quotes around curly braces when embedding a JavaScript expression in an attribute. You should either use quotes (for string values) or curly braces (for expressions), but not both in the same attribute.
+Zet geen aanhalingstekens om accolades als je een JavaScript expressie wilt gebruiken in een attribuut. Je gebruikt òf aanhalingstekens (voor string waarden) òf accolades (voor expressies), maar niet beide in hetzelfde attribuut.
 
->**Warning:**
+>**Waarschuwing:**
 >
->Since JSX is closer to JavaScript than to HTML, React DOM uses `camelCase` property naming convention instead of HTML attribute names.
+>Aangezien JSX meer op JavaScript lijkt dan op HTML, gebruikt React DOM de `camelCase` property naamgevingsconventie in plaats van HTML attribuutnamen.
```

Comment:
Maybe "conventie van naamgeving" is more readable than "naamgevingsconventie"? 🤔

Reply:
Not sure, it doesn't feel like Dutch to me. We need a third opinion :).

Reply:
Tagging @jochenboesmans and @rickbeerendonk then :)

Reply:
Personally I have no problem with "naamgevingsconventie". In fact, I think it sounds a bit more fluent. However, "conventie van naamgeving" might indeed be more appropriate since we're trying to make the text as easy to understand as possible. I'm fine with either.

Reply:
Googling on [naamgevingsconventie](https://www.google.com/search?q=naamgevingsconventie)  provides less programming related links compared to [conventie van naamgeving](https://www.google.com/search?q=conventie+van+naamgeving). People that understand what it mean won't have problem with either, but inexperienced developers will most likely Google it, so personally my preference goes towards "Conventie van naamgeving"